Lombiq.DataTables.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -124,4 +170,24 @@ private static IEnumerable<object> AdjustForProvider(object[] source) =>
                 ((string)source[^1]).Replace(",", string.Empty),
                 null,
             ]);
+
+    private static async Task DownloadSpreadsheetAsync(
+        this UITestContext context,
+        By downloadButtonBy,
+        int expectedLength)
+    {
+        var path = context.GetTempSubDirectoryPath(DirectoryPaths.Downloads, "export.xlsx");
```

Comment:
A shortcut for `GetTempSubDirectoryPath(DirectoryPaths.Downloads, fileName` in `UITestContext` would be useful, so you don't have to go a level deeper with `GetTempSubDirectoryPath` after having access to `DownloadsDirectoryPath`.